docs/UserGuide.md

```diff
@@ -319,6 +305,22 @@ Updates the amount of tuition fee paid by the student and the amount of tuition
 
 <div style="page-break-after: always;"></div>
 
+### Showing income data: `income`
+
+Shows the total amount of tuition fee that you have been paid, as well as total amount that is owed.
```

Comment:
as well as **the** total amount right?

Reply:
Thanks for the suggestion! Ive made the changes